sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityInterface.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.sensitivity_computation;
+
+import com.farao_community.farao.commons.FaraoException;
+import com.farao_community.farao.commons.Unit;
+import com.farao_community.farao.data.crac_api.Cnec;
+import com.farao_community.farao.data.crac_api.Crac;
+import com.farao_community.farao.data.crac_api.RangeAction;
+import com.farao_community.farao.flowbased_computation.glsk_provider.GlskProvider;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.sensitivity.SensitivityComputationParameters;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Objects;
+import java.util.Set;
+
+
+/**
+ * An interface with the engine that computes sensitivities and flows needed in the RAO.
+ *
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ * @author Philippe Edwards {@literal <philippe.edwards at rte-france.com>}
+ * @author Baptiste Seguinot {@literal <baptiste.seguinot at rte-france.com>}
+ */
+public final class SystematicSensitivityInterface {
```

Comment:
You should not make it final

Reply:
yeah not sure why i did that, probably copying code from elsewhere :shrug:

Reply:
nvm, all the constructors are private that's why i did it